app/src/editor/shared/components/modules/Chart.jsx

```diff
@@ -5,78 +5,105 @@ import useIsMounted from '$shared/hooks/useIsMounted'
 import Chart from '$editor/shared/components/Chart'
 import ModuleSubscription from '$editor/shared/components/ModuleSubscription'
 import UiSizeConstraint from '../UiSizeConstraint'
+import useOnceEffect from '$shared/hooks/useOnceEffect'
+
+const seriesId = (idx) => (
+    `series-${idx}`
```

Comment:
why

Reply:
Don't know. Copied it over, modded and reused. XD Will dig deeper. 👌

Reply:
Maybe I did this, might have been so that it was clear when it was a series id vs some other id